utils/timeout_blacklist.py

```diff
@@ -0,0 +1,20 @@
+from datetime import timedelta
+
+import redis
+
+TIMEOUT_TOLERANCE = 3
+EXPIRATION_TIME = timedelta(minutes=30)
+user_connection = redis.Redis(host='redis-10060.c135.eu-central-1-1.ec2.cloud.redislabs.com', port=10060, username='default', password='CWqOy4UlreQiOgo8lenpSc6ZMnCvTA8x', decode_responses=True)
```

Comment:
not sure where to put those credentials

Reply:
It's ok, we are knowingly taking a risk here, this details will be in our repository.
there is an interesting task(you will not do it this sprint) that is about storing secrets on git which will fix it.
@Hazannovich

Reply:
thank you! I see that there is a choice to put environment secrets  or repository secrets or organization secrets:
https://docs.github.com/en/actions/security-guides/encrypted-secrets
will read more about the difference between them but If can you just tell me if only the maintainer can set those up? and if in our case its environment secrets right? and if it is where i should put the flask app.config values like URL_TIMEOUT and SECRET_KEY?

Reply:
I will be surprised if this service is free.
you are welcome to check it out. but anyway- it's not included in the scope of this ticket. 
But I must say I am curious about this option I will check, but anyway, the current technology that I wanted to use is SOPS which is very common in the industry and worth the learn about it.
welcome to check the ticket 
https://bear-playground.atlassian.net/browse/PLAYG-33